classes/class-gf-securesubmit.php

```diff
@@ -1688,9 +1678,9 @@ private function buildCardHolder($feed, $submission_data, $entry)
      */
     private function buildCheckHolder($feed, $submission_data, $entry)
     {
-        $checkHolder = new HpsCheckHolder();
+        $checkHolder = new ECheck();
         $checkHolder->address = $this->buildAddress($feed, $submission_data, $entry);
-        $checkHolder->checkName = htmlspecialchars(rgar($submission_data, 'ach_check_holder')); //'check holder';
+        $checkHolder->checkHolderName = htmlspecialchars(rgar($submission_data, 'ach_check_holder')); //'check holder';
```

Comment:
This should still be `checkName`. The `checkHolderName` property isn't mapped to the correct gateway field.

Reply:
updated as per review comment